reporters/cloud-run-scripts/SendSlackReport.sh

```diff
@@ -28,7 +28,7 @@ pathToFile="./"$fileName
 # Send a curl command to upload the file along with a message to the channel. Store response log for debugging.
 responseFileName="response.json"
 echo "Sending file as slack message"
-curl https://slack.com/api/files.upload -F token="${SLACK_TOKEN}" -F channels="${SLACK_CHANNEL}" -F title="${message_title}" -F fileName="${fileName}" -F file=@"${pathToFile}" | jq > $responseFileName
+curl https://slack.com/api/files.upload -F token="${SLACK_TOKEN}" -F channels="${SLACK_CHANNEL}" -F title="${messageTitle}" -F fileName="${fileName}" -F file=@"${pathToFile}" | jq '.' > $responseFileName
```

Comment:
this little bugger after the `jq` took a good hour to debug: mac's version of `js` and Linux have different default behavior. On Mac, if you do not specify any params then `jq` automatically spreads the object (same effect as `'.'`). However, this does not apply in Linux. Ended up having to run the docker container locally and manually play around with sending malformed messages until this worked.

Reply:
great catch, I should have been more explicit in my original impl.!